user/api.go

```diff
@@ -23,12 +23,12 @@ func Update(ctx context.Context, id string, params *UpdateParams) (*clerk.User,
 	return getClient().Update(ctx, id, params)
 }
 
-// UpdateProfileImage sets or replaces the users's profile image.
+// UpdateProfileImage sets or replaces the user's profile image.
 func UpdateProfileImage(ctx context.Context, id string, params *UpdateProfileImageParams) (*clerk.User, error) {
 	return getClient().UpdateProfileImage(ctx, id, params)
 }
 
-// DeleteProfileImage removes the users's profile image.
+// DeleteProfileImage sets or replaces the user's profile image.
```

Comment:
I might be missing some context here on that feature, but why a `Delete` method would `set or replace`?

Reply:
Oh, that was from `go gen ./...`, It generates this api file based on the methods from the other clients, I'll fix those and re-run it! Thanks for spotting that

Reply:
resolved via [5548cc3](https://github.com/clerk/clerk-sdk-go/pull/322/commits/5548cc3627c559f138581078390db4a96d4a319e)